tests/spread/integration/iproute2/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# smoketest ip commands
+ip --help | grep "Usage:"
+
+# test some basic commands and against loopback that
+# we can kinda expect is there
+ip link | grep "LOOPBACK,UP"
```

Comment:
can you please also test the other binaries being installed? I know it might be tedious, but that's the only way we can ensure the slice has all it needs for them to behave properly